### DISCUSSION TITLE
Can I use StereoKit with Meta Quest 3?

### DISCUSSION THREAD

**Discussion by llfab** [StereoKit/StereoKit#870] [Q&A] [answered]
Seems that Meta Quest 3 uses OpenXR. However, I have not seen it as supported device. How about it?

**Comment by mvvvv** [comment, marked as answer]
It works for all Quest yes. You can see a nice demo there : https://www.youtube.com/watch?v=of4qh9O9DWE

**Reply by llfab** [reply]
Thanks for the quick reply!
